cli/schema/cypress.schema.json

```
@@ -278,6 +278,76 @@
           "type": "boolean",
           "default": false,
           "description": "Enables including elements within the shadow DOM when using querying commands (e.g. cy.get(), cy.find()). Can be set globally in cypress.json, per-suite or per-test in the test configuration object, or programmatically with Cypress.config()"
+        },
+        "clientPkiCertificates": {
+          "description": "Defines client PKI certificates to use when sending requests to the specified URLs",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "url": {
+                "description": "URL matching string",
+                "type": "string"
+              },
+              "ca": {
+                "description": "File path(s) (absolute or relative) to CA files to validate certs against",
```

Generally we want to discourage users to hard-code absolute paths in their projects, as it makes it harder for users to share tests with other developers.

```suggestion
                "description": "Path(s) to CA file(s) to validate certs against, relative to project root.",
```